fluent-web/examples/webcomponent.html

```diff
@@ -71,6 +71,8 @@
       </div>
     </template>
 
-    <fluent-widget data-l10n-id="proceed-button"></fluent-widget>
+    <fluent-widget
+      data-l10n-id="proceed-button"
+      data-l10n-attrs="label, action-ok, action-cancel"></fluent-widget>
```

Comment:
`class` uses space-separated lists of values, but the newer `srcset` and `sizes` both use comma-separated, so this is fine. (I recall being against commas at some point.)